### DISCUSSION TITLE
Jest safeDescriptors: false property does not exist

### DISCUSSION THREAD

**Discussion by adovlyash** [mobxjs/mobx#2783] [Q&A] [answered]
Hi all,

I'm using computed value in my MobX v6.1.5 store like this (I've shortened implementation for simplicity):

```
import { observable, action, makeObservable, computed } from 'mobx';

class DrawerStore {
    @observable cursorMode: string = 'Line';

    constructor() {
        makeObservable(this);
    }

    @computed get isHighlightEnabled(): boolean {
        return this.cursorMode === 'Arrow';
    }
}

export default new DrawerStore();
```

And now I want to test some functionality which uses this store via jest. I've configured jest like this https://mobx.js.org/configuration.html#safedescriptors-boolean and now I want to use spy for drawerStore.isHighlightEnabled:

```
import drawerStore from 'store/drawerStore';
jest.mock('store/drawerStore');

...
jest.spyOn(drawerStore, 'isHighlightEnabled', 'get').mockReturnValue(true);
...

```
But I get error: **isHighlightEnabled property does not exist**

Can anyone please suggest if this functionality works now and maybe provide some example of working tests with spies for store getters?

**Comment by mweststrate** [comment]
Cause you named it `isHighlightEnabled` not, `isHighlight` :)?

**Reply by adovlyash** [reply]
Sorry. it's a typo. The name in tests is "isHighlightEnabled" too

**Reply by adovlyash** [reply]
![image](https://user-images.githubusercontent.com/10262583/107016973-a29bf680-67af-11eb-89f0-376307ac1f62.png)

**Comment by urugator** [comment, marked as answer]
https://github.com/facebook/jest/issues/9675

**Reply by adovlyash** [reply]
Hmmm. So it's a jest issue. Thank you!
